packages/cli-dxns/src/dxos.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package dxos;
+
+import "google/protobuf/any.proto";
+
+message Kube {
+  string name = 1;
+  string url = 2;
+  string version = 3;
+}
+
+message Service {
+  string name = 1;
+  string type = 2;
+  string version = 3;
+
+  // Possibility of inheritance for services?
+  google.protobuf.Any service = 4;
+}
+
+message App {
+  string name = 1;
+  optional string description = 2;
+  string version = 3;
+  optional string author = 4;
+  string build = 5;
+  // IPFS CID.
+  string package = 6;
```

Comment:
let's be more explicity e.g., binary_hash?
also need platform type and version (e.g, NodeJS, 14) (maybe platform message)